src/web/controllers/admin/access_admin_controller.py

```diff
@@ -320,24 +275,44 @@ def _validate_account_form_data(
         errors: dict[str, str] = {}
         event = web_context.get_admin_event()
         account = web_context.admin_account
-        if not (account and account.anonymous):
-            username = WebContext.form_data_to_str(data, field := 'username')
-            if not username:
-                errors[field] = _('Please enter the username.')
-            elif not re.match(r'^[a-zA-Z0-9_\-]+$', username):
+        if not account or account.user_account:
+            first_name: str = (
+                WebContext.form_data_to_str(data, field := 'first_name') or ''
+            )
+            if first_name and not re.match(r"^[a-zA-Z0-9'\-]*$", first_name):
                 errors[field] = _(
-                    'Accepted characters are letters, numbers, underscore (_) and minus (-).'
+                    "Accepted characters are letters, numbers, single quote (') and minus (-)."
                 )
-            elif username in event.accounts_by_username and (
-                action != FormAction.UPDATE
-                or (account and account.username != username)
-            ):
-                errors[field] = _('Account [{username}] already exists.').format(
-                    username=username
+            last_name: str = (
+                WebContext.form_data_to_str(data, field := 'last_name') or ''
+            )
+            if not last_name:
+                errors[field] = _('Please enter the last name.')
```

Comment:
```suggestion
                errors[field] = _('This field is required.')
```
It's the kind of message that sounds cleaner and are more instinctive to users when they are generic and not customized (ex: the user sees the message and automatically knows without reading what the problem is, without even reading).

It is also easier to maintain / to consider a new translation to not have all of these customized messages for no reason